tests/unit/test_dask_nvt.py

```diff
@@ -163,6 +163,8 @@ def op_logic(self, *args, **kwargs):
     assert math.isclose(result.id.max(), processor.stats["maxs"]["id"], rel_tol=1e-3)
 
 
+# This test fails on known issue https://github.com/rapidsai/cudf/issues/5322
```

Comment:
I merged a fix for this bug a week or two ago - Should work fine with nightly?

Reply:
We dont have that in CI, working on getting it in currently.